src/modules/ROOT/pages/changelog/202303/05.json

```diff
@@ -5,10 +5,10 @@
   "added": [
     {
       "what": [
-        "the option to configure whether module field value is on the same row as its label in the record configurator."
+        "page navigation to stop loading previous resources in case we've navigated to a new page."
```

Comment:
Shouldn't this stay as changed? Lapsus?

Reply:
just the record config part should be changed to page block as you wrote a couple of comments up. I pasted the wrong contents in the wrong place